src/main.dart

```diff
@@ -1,5 +1,183 @@
-import 'core/note_app.dart';
+import 'dart:io';
 
+class Note {
+  late String title;
+  late String content;
+
+  Note({
+    required this.title,
+    required this.content,
+  });
+}
+
+//Manage operation of program (create,edit,delete,search,display).
+class NoteManager{
+  List <Note> notes=[];
+
+
+//Creates a new note and adds it to the list of notes.
+ void createNote(){
+  String title,content;
+  print("Enter the note title: ");
+  title=stdin.readLineSync() ?? "None";
+
+  print("Enter the note content: ");
+  content=stdin.readLineSync() ?? "None";
+
+  Note newNote=Note(title:title,content:content);
+  notes.add(newNote);
+
+  print("The note is successfully created!");  
+  }
+
+  //Display all notes of the list. 
+  void DisplayNote(){
+    if(notes.isEmpty){
+      print("Nothing to display!");
+    }
+    else{
+      print("All notes: ");
+      for(var note in notes)
+        print("Title: ${note.title}\n Contents: ${note.content}\n ");
+
+    }  
+  }
+
+  //Edit the title and content of note. 
+  void EditNote(){
```

Comment:
Ensure that you follow the Dart conventions by using lowerCamelCase when naming your functions.

editNote() start with small letter